### PR TITLE
Fix a couple of mobile display issues

### LINKED ISSUE #3699: Improve `store settings` view alignment on mobile [iPhone 8]
Just noticed that the Store Settings view isn't well aligned on iPhone 8 for some reason, would be nice if we could fix this.

https://user-images.githubusercontent.com/36959754/166134812-b7b106da-5614-48c9-bfc0-b7c3f64a3a3b.MOV

### PR DESCRIPTION
**Pull request by bolatovumar**
|Before|After|
|---|---|
|Capture|Capture2|

Close #3699

### CODE REVIEW

File: BTCPayServer/Views/UIStores/GeneralSettings.cshtml [hunk 1]
```diff
@@ -45,11 +45,14 @@
                 <a href="https://docs.btcpayserver.org/FAQ/Stores/#add-network-fee-to-invoice-vary-with-mining-fees" target="_blank" rel="noreferrer noopener">
                     <span class="fa fa-question-circle-o text-secondary" title="More information..."></span>
                 </a>
-                <select asp-for="NetworkFeeMode" class="form-select w-auto">
-                    <option value="MultiplePaymentsOnly">... only if the customer makes more than one payment for the invoice</option>
-                    <option value="Always">... on every payment</option>
-                    <option value="Never">Never add network fee</option>
-                </select>
+                @* For whatever reason wrapping the select with this div fixes this Safari bug: https://github.com/btcpayserver/btcpayserver/issues/3699 *@
```
Comment (bolatovumar) [inline BTCPayServer/Views/UIStores/GeneralSettings.cshtml:48]: I don't understand why this fixes the issue but it does. I was able to repro the bug in Safari only, this issue doesn't reproduce in Chrome. Also, it's intermittent in Safari but once it happens I can reproduce it consistently but sometimes it stops happening all of a sudden. I think it's related to changing zoom levels.

File: BTCPayServer/Views/UIServer/Services.cshtml [hunk 2]
```diff
@@ -117,7 +117,7 @@
             {
                 <tr>
                     <td>@s.Name</td>
-                    <td style="word-wrap: anywhere;">
```
Comment (bolatovumar) [inline BTCPayServer/Views/UIServer/Services.cshtml:120]: `word-wrap: anywhere;` doesn't actually do anything. `anywhere` is not a valid value: https://www.w3schools.com/cssref/css3_pr_word-wrap.asp

Comment (NicolasDorier) [reply BTCPayServer/Views/UIServer/Services.cshtml:120]: ping @dennisreimann

Comment (dennisreimann) [reply BTCPayServer/Views/UIServer/Services.cshtml:120]: The `text-break` class should work here ([details(https://getbootstrap.com/docs/5.2/utilities/text/#word-break)).

File: BTCPayServer/Views/UIStores/GeneralSettings.cshtml (hunk 1 above)
Comment (dennisreimann) [reply BTCPayServer/Views/UIStores/GeneralSettings.cshtml:48]: I bet it is caused by the text in `MultiplePaymentsOnly` being longer than the width of the content allows for.

**Pull request merged**